docs/get-started/getting-started-gcp.md

```diff
@@ -244,13 +252,20 @@ metadata parameter `MIG_CGI`. Below is an example of using a profile name and a
 ```
 
 This may take around 10-15 minutes to complete.  You can navigate to the Dataproc clusters tab in
-the Google Cloud Console to see the progress.
+the Google Cloud Console to see the progress.  
 
 ![Dataproc Cluster](../img/GCP/dataproc-cluster.png)
 
 If you'd like to further accelerate init time to 4-5 minutes, create a custom Dataproc image using
 [this](#build-custom-dataproc-image-to-accelerate-cluster-init-time) guide. 
 
+### Cluster creation troubleshooting
+If you encounter an error related to GPUs not being available because of your account quotas, please 
+go to this page for updating your quotas: [Quotas and limits](https://cloud.google.com/compute/quotas).
+
+If you encounter an error related to GPUs not available in the specific region or zone, you will 
+need to update the REGION or ZONE parameter in the cluster creation command.
```

Comment:
is there any case they are temporarily unavailable and we should state try back later?  I realize that isn't ideal though.

Reply:
It is possible and GCP error message could state that explicitly, e.g. 
```Operation type [insert] failed with message "The zone 'projects/xxxxxx/zones/us-central1-a' does not have enough resources available to fulfill the request. Try a different zone, or try again later." ```

I'd rather point to updating REGION or ZONE as that is the specific limit issue that I've seen in practice on Dataproc.